sample-config.ini

```diff
@@ -160,3 +160,8 @@
 # Default: True
 #
 # showOlderEdits = True
+
+# Whether to show the indicator for users typing.
+# Default: False
```

Comment:
Can you also indicate here that this also controls whether typing notifications are *sent*, too?